docs/help-center/docs/warden-app/fees.md

```
@@ -23,7 +23,16 @@ All swaps incur these fees:
 | Fee type      | Amount | Currency           | Description                                              |
 | ------------- | -------|--------------------| -------------------------------------------------------- |
 | Gas fee       | varies | native coin        | A fee paid to the network for processing the transaction |
-| Platform fee  | 0.85%  | input/output token | A percentage charged by Warden                           |
+| Platform fee  | 0.85%  | input/output token | A percentage charged by Warden     
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)